.gitignore

```diff
@@ -6,3 +6,4 @@ verify.js.map
 verify.js
 flattened.*.sol
 deployment-state.json
+contracts-optimized
```

Comment:
why is that added here?

Reply:
This update (the optimize feature) was added after the last update of the migration experimental, and therefore was not a part of it. I rebased the package on master so it will have those later features.